fakeData.js

```diff
@@ -0,0 +1,93 @@
+const psychiarists = [
+    {
+        doctorImg: "https://images.all-free-download.com/images/graphiclarge/doctor_work_painting_colored_cartoon_character_sketch_6847622.jpg",
+        name: "Dr. Nabeel No'man",
+        specialtyies: "Anxiety, Bipolar Disorder, Depression, General Adult Psychiatry, Personality Disorders",
+        workplace: "Azal Hospital",
+        address: "60th street",
+        email: "example@gmail.com",
+        phone: "+967770000000"
+    },
+{
+        doctorImg: "https://images.all-free-download.com/images/graphiclarge/doctor_work_painting_colored_cartoon_character_sketch_6847622.jpg",
+        name: "Dr. Nabeel No'man",
+        specialtyies: "Anxiety, Bipolar Disorder, Depression, General Adult Psychiatry, Personality Disorders",
+        workplace: "Azal Hospital",
+        address: "60th street",
+        email: "example@gmail.com",
+        phone: "+967770000000"
+    },
+  {     
+        doctorImg: "https://images.all-free-download.com/images/graphiclarge/doctor_work_painting_colored_cartoon_character_sketch_6847622.jpg",
+        name: "Dr. Nabeel No'man",
+        specialtyies: "Anxiety, Bipolar Disorder, Depression, General Adult Psychiatry, Personality Disorders",
+        workplace: "Azal Hospital",
+        address: "60th street",
+        email: "example@gmail.com",
+        phone: "+967770000000"
+    },
+  {
+        doctorImg: "https://images.all-free-download.com/images/graphiclarge/doctor_work_painting_colored_cartoon_character_sketch_6847622.jpg",
+        name: "Dr. Nabeel No'man",
+        specialtyies: "Anxiety, Bipolar Disorder, Depression, General Adult Psychiatry, Personality Disorders",
+        workplace: "Azal Hospital",
+        address: "60th street",
+        email: "example@gmail.com",
+        phone: "+967770000000"
+    },
+  {
+        doctorImg: "https://images.all-free-download.com/images/graphiclarge/doctor_work_painting_colored_cartoon_character_sketch_6847622.jpg",
+        name: "Dr. Nabeel No'man",
+        specialtyies: "Anxiety, Bipolar Disorder, Depression, General Adult Psychiatry, Personality Disorders",
+        workplace: "Azal Hospital",
+        address: "60th street",
+        email: "example@gmail.com",
+        phone: "+967770000000"
+    }
+];
+const articles = [
+    {
```

Comment:
We need to add date here since it is displayed in the Article page and in home page.